commons/src/main/kotlin/com/simplemobiletools/commons/activities/LicenseActivity.kt

```diff
@@ -3,25 +3,69 @@ package com.simplemobiletools.commons.activities
-import com.simplemobiletools.commons.helpers.*
+import com.simplemobiletools.commons.databinding.ActivityLicenseBinding
+import com.simplemobiletools.commons.databinding.ItemLicenseBinding
+import com.simplemobiletools.commons.extensions.getProperBackgroundColor
+import com.simplemobiletools.commons.extensions.getProperPrimaryColor
+import com.simplemobiletools.commons.extensions.getProperTextColor
+import com.simplemobiletools.commons.extensions.launchViewIntent
+import com.simplemobiletools.commons.extensions.updateTextColors
+import com.simplemobiletools.commons.extensions.viewBinding
+import com.simplemobiletools.commons.helpers.APP_ICON_IDS
+import com.simplemobiletools.commons.helpers.APP_LAUNCHER_NAME
+import com.simplemobiletools.commons.helpers.APP_LICENSES
+import com.simplemobiletools.commons.helpers.LICENSE_ANDROID_LAME
+import com.simplemobiletools.commons.helpers.LICENSE_APNG
+import com.simplemobiletools.commons.helpers.LICENSE_AUDIO_RECORD_VIEW
+import com.simplemobiletools.commons.helpers.LICENSE_AUTOFITTEXTVIEW
+import com.simplemobiletools.commons.helpers.LICENSE_CROPPER
+import com.simplemobiletools.commons.helpers.LICENSE_ESPRESSO
+import com.simplemobiletools.commons.helpers.LICENSE_EVENT_BUS
+import com.simplemobiletools.commons.helpers.LICENSE_EXOPLAYER
+import com.simplemobiletools.commons.helpers.LICENSE_FILTERS
+import com.simplemobiletools.commons.helpers.LICENSE_GESTURE_VIEWS
+import com.simplemobiletools.commons.helpers.LICENSE_GIF_DRAWABLE
+import com.simplemobiletools.commons.helpers.LICENSE_GLIDE
+import com.simplemobiletools.commons.helpers.LICENSE_GSON
+import com.simplemobiletools.commons.helpers.LICENSE_INDICATOR_FAST_SCROLL
+import com.simplemobiletools.commons.helpers.LICENSE_JODA
+import com.simplemobiletools.commons.helpers.LICENSE_KOTLIN
+import com.simplemobiletools.commons.helpers.LICENSE_LEAK_CANARY
+import com.simplemobiletools.commons.helpers.LICENSE_M3U_PARSER
+import com.simplemobiletools.commons.helpers.LICENSE_NUMBER_PICKER
+import com.simplemobiletools.commons.helpers.LICENSE_OTTO
+import com.simplemobiletools.commons.helpers.LICENSE_PANORAMA_VIEW
+import com.simplemobiletools.commons.helpers.LICENSE_PATTERN
+import com.simplemobiletools.commons.helpers.LICENSE_PDF_VIEWER
+import com.simplemobiletools.commons.helpers.LICENSE_PDF_VIEW_PAGER
+import com.simplemobiletools.commons.helpers.LICENSE_PHOTOVIEW
+import com.simplemobiletools.commons.helpers.LICENSE_PICASSO
+import com.simplemobiletools.commons.helpers.LICENSE_REPRINT
+import com.simplemobiletools.commons.helpers.LICENSE_ROBOLECTRIC
+import com.simplemobiletools.commons.helpers.LICENSE_RTL
+import com.simplemobiletools.commons.helpers.LICENSE_SANSELAN
+import com.simplemobiletools.commons.helpers.LICENSE_SMS_MMS
+import com.simplemobiletools.commons.helpers.LICENSE_STETHO
+import com.simplemobiletools.commons.helpers.LICENSE_SUBSAMPLING
+import com.simplemobiletools.commons.helpers.LICENSE_ZIP4J
```

Comment:
guess we should use an asterisk at importing these, as previously

Reply:
It's a bad development practice to use asterisk import

Reply:
lets use asterisks after 5 imports, it is written in editorconfig file too